### PR TITLE
Update me0Digi validation

### PR DESCRIPTION
**Pull request by calabria**
Adding plots for the expected background rates (given the results from FLUKAxGEANT). Update the binning to match the me0 8 eta-partition geometry.
@kpedro88

### CODE REVIEW

File: Validation/MuonME0Validation/src/ME0DigisValidation.cc [hunk 1]
```diff
@@ -36,12 +36,39 @@ void ME0DigisValidation::bookHistograms(DQMStore::IBooker & ibooker, edm::Run co
...
+  for(int i = 1; i < me0_strip_exp_bkgNeutral_rad->getTH1F()->GetSize(); i++){
+      
+    double pos = me0_strip_exp_bkgNeutral_rad->getTH1F()->GetBinCenter(i);
+    double neutral = 0;
+    double charged = 0;
+      
+    double pos_helper = 1.0;
+    for(int j=0; j<7; ++j) {neutral += neuBkg[j]*pos_helper; pos_helper *= pos;}
+    pos_helper = 1.0;
+    for(int j=0; j<7; ++j) {charged += eleBkg[j]*pos_helper; pos_helper *= pos;}
```
Comment (kpedro88) [inline Validation/MuonME0Validation/src/ME0DigisValidation.cc]: Couldn't this loop be combined with the previous loop?

**Comment by kpedro88** [comment]
please test

**Comment by kpedro88** [comment]
@cmsbuild please test

**Comment by dmitrijus** [comment]
+1

File: Validation/MuonME0Validation/plugins/MuonME0DigisHarvestor.cc [hunk 2]
```diff
@@ -119,7 +119,7 @@ TH1F* MuonME0DigisHarvestor::ComputeBKG(TH1F* hist1, TH1F* hist2, std::string na
     }
 
     int nEvts = hist2->GetEntries();
-    float scale = 6*2*nEvts*9*25e-9;
+    float scale = 6*2*nEvts*3*25e-9;
```
Comment (davidlange6) [inline Validation/MuonME0Validation/plugins/MuonME0DigisHarvestor.cc]: Hi @calabria - please add a comment to explain this set of magic numbers

Comment (calabria) [inline Validation/MuonME0Validation/plugins/MuonME0DigisHarvestor.cc]: 6 layers, 2 endcap regions, we are saving hits only from 3 BXs (-1,0,+1) of 25 ns (before it was 9)

Comment (davidlange6) [inline Validation/MuonME0Validation/plugins/MuonME0DigisHarvestor.cc]: I meant in the code:) Thanks for adding it
[but can "3" be deduced from some other source of information?]

Comment (calabria) [inline Validation/MuonME0Validation/plugins/MuonME0DigisHarvestor.cc]: at the moment this information can be taken from here https://github.com/cms-sw/cmssw/blob/master/SimMuon/GEMDigitizer/python/muonME0ReDigis_cfi.py#L11 , when we move to the realistic digitizer from similar lines

**Comment by calabria** [comment]
if there are no further comments can we integrate this PR? Thanks.

**Comment by davidlange6** [comment]
it is need of adding a comment to the code to explain the line discussed above

**Comment by dmitrijus** [comment]
+1

**Pull request merged**